### ISSUE TITLE
[Improvement] use lsof or similar to determine which processes need to be handled by STOP/CONT

### ISSUE THREAD

**Issue by hg42** [machiav3lli/oandbackupx#398]
* only processes [EDIT: not belonging to the app user] that actually use files in app directories need to be stopped
  [EDIT: note, all processes belonging to the app should still be stopped, because they could write later and they could also use services that write data]
* lsof can be used to determine open files in app directories

but
* there are probably race conditions (but using UID isn't much better, detection/stopping are not atomic in both cases)
* lsof output seems to be unexpected in some cases
* lsof may be difficult to parse

**Comment by hg42** [comment]
an example:
```
root@RMX1993L1:/ # lsof | grep /data/data/com.google.android.gm | awk '{print $1,$2}' | sort -u                                                                
.android.chrome 1131
.apps.messaging 14215
.gms.persistent 1387
earchbox:search 11501
gle.android.gms 26744
id.gms.unstable 17515
ocessService0:0 23293
ogle.android.gm 24420
```
```
root@RMX1993L1:/ # ps -o PID,USER,NAME -u u0_a316     
  PID USER         NAME                       
24420 u0_a316      com.google.android.gm
```

so there are (sometimes) a lot more processes with open files in `/data/data/PKGNAME`

**Comment by hg42** [comment]
it looks like we could use a command like
`lsof | grep /data/data/com.google.android.gm | awk '{print $2}' | sort -u`
to determine the PIDs.

It's probably better to use the same list for STOP and CONT (not determine it again on postprocessPackage, so we should remember them, which could also handle the reference counting, see #397

**Comment by hg42** [comment]
I'll close this, because it's included in #414
